Gems/Atom/Feature/Common/Code/Source/AuxGeom/AuxGeomDrawQueue.cpp

```diff
@@ -584,6 +585,176 @@ namespace AZ
+                        FarBottomRight, FarBottomLeft,
+                        FarBottomLeft, FarTopLeft,
+
+                        //Near-to-Far connecting lines
+                        NearTopLeft, FarTopLeft,
+                        NearTopRight, FarTopRight,
+                        NearBottomLeft, FarBottomLeft,
+                        NearBottomRight, FarBottomRight,
+                    };
+
+                    RPI::AuxGeomDraw::AuxGeomDynamicIndexedDrawArguments drawArgs;
+                    drawArgs.m_verts = corners.data();
+                    drawArgs.m_vertCount = 8;
+                    drawArgs.m_indices = lineIndices;
+                    drawArgs.m_indexCount = 24;
+                    drawArgs.m_colors = &color;
+                    drawArgs.m_colorCount = 1;
+                    drawArgs.m_depthTest = depthTest;
+                    drawArgs.m_depthWrite = depthWrite;
+                    drawArgs.m_viewProjectionOverrideIndex = viewProjOverrideIndex;
+                    DrawLines(drawArgs);
+
+                    if (style == DrawStyle::Solid || style == DrawStyle::Shaded)
+                    {
+                        // DrawTriangles doesn't support shaded drawing, so we can't support it here either.
+                        AZ_WarningOnce("AuxGeomDrawQueue", style != DrawStyle::Shaded, "Cannot draw frustum with Shaded DrawStyle, using Solid instead.");
+
+                        uint32_t triangleIndices[36]{
+                            //near
+                            NearBottomLeft, NearTopLeft, NearTopRight,
+                            NearBottomLeft, NearTopRight, NearBottomRight,
+
+                            //far
+                            FarBottomRight, FarTopRight, FarTopLeft,
+                            FarBottomRight, FarTopLeft, FarBottomLeft,
+
+                            //left
+                            NearTopLeft, NearBottomLeft, FarBottomLeft,
+                            NearTopLeft, FarBottomLeft, FarTopLeft,
+
+                            //right
+                            NearBottomRight, NearTopRight, FarTopRight,
+                            NearBottomRight, FarTopRight, FarBottomRight,
+
+                            //bottom
+                            FarBottomLeft, NearBottomLeft, NearBottomRight,
+                            FarBottomLeft, NearBottomRight, FarBottomRight,
+
+                            //top
+                            NearTopLeft, FarTopLeft, FarTopRight,
+                            NearTopLeft, FarTopRight, NearTopRight,
+                        };
+
+                        Color transparentColor(color.GetR(), color.GetG(), color.GetB(), color.GetA() * 0.3f);
+                        drawArgs.m_indices = triangleIndices;
+                        drawArgs.m_indexCount = 36;
+                        drawArgs.m_colors = &transparentColor;
+                        drawArgs.m_opacityType = OpacityType::Translucent;
```

Comment:
This will be ignored, DrawTriangles uses the color alpha to determine the opacity type.